frontend/src/components/bs/bs.tsx

```diff
@@ -8,6 +8,21 @@ import { bs as BSActions } from 'store/actions';
 
 const BS: FC = () => {
   const dispatch = useAppDispatch();
+  const changeHandler = (e: React.FormEvent<HTMLInputElement>) => {
+    const formData = new FormData();
+    const files = e.currentTarget.files;
+    if (files) {
+      formData.append('file', files[0]);
+    }
+    const spaceId = '5148d1ab-3929-4403-8922-f5d86cddbc11';
+    fetch(`http://localhost:3001/api/v1/bs/spaces/${spaceId}/objects`, {
+      method: 'POST',
+      headers: {
+        authorization: `Bearer ${localStorage.getItem('token')}`,
+      },
+      body: formData,
+    }).then((response) => response.json());
+  };
```

Comment:
Move this code to service

Reply:
I left the front part for testing, when the edits are over I will remove it.